crates/holochain/src/core/workflow/app_validation_workflow.rs

```diff
@@ -438,8 +437,8 @@ pub async fn validate_op(
     let network = Arc::new(network.clone());
 
     let zomes_to_invoke = get_zomes_to_invoke(op, &workspace, network.clone(), ribosome).await;
-    if let Err(OutcomeOrError::Err(e)) = &zomes_to_invoke {
-        tracing::error!("Error getting zomes to invoke: {e} to validate op {op:?}");
+    if let Err(OutcomeOrError::Err(err)) = &zomes_to_invoke {
+        tracing::error!("Error getting zomes to invoke to validate op {op:?}: {err}");
```

Comment:
This is also a nice option instead of having to shove errors into strings

```suggestion
        tracing::error!(?op, err, "Error getting zomes to invoke to validate op");
```

Reply:
Ah, yes, I haven't used that. What's the difference?